src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+unreleased
++++++++++++++
+* Deprecate --orchestrator-release option in acs create
```

Comment:
This should be under 2.0.19.
We don't require `unreleased` any more and CI is failing because of this.

The change log goes under the *next* version of the module we'll release.
Since [2.0.18](https://pypi.python.org/pypi/azure-cli-acs) is what is published, your change goes under 2.0.19 and that heading is already there from a prev. PR.